components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Runtime/Default/SecurityConfig.cs

```diff
@@ -17,6 +17,7 @@ public class SecurityConfig : ConfigBase, ISecurityConfig {
         /// Configuration
         /// </summary>
 #pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+        public const string PkiRootPathKey = "PkiRootPath";
```

Comment:
>PkiRootPath [](start = 46, length = 11)

i think it was "pki" in the stack. maybe they have changed.

Reply:
the default pki root folder is "pki" indeed - I think is a spec recommendation. This is just the config key string